mdx-models/src/main/java/com/mx/path/model/mdx/model/authorization/Authorization.java

```diff
@@ -16,117 +20,10 @@ public class Authorization extends MdxBase<Authorization> {
   private String deviceId;
   private Long expiresAt;
   private NameValuePair[] headers;
-  private JavaScript javascript;
+  private JavaScript javaScript;
```

Comment:
Could this respelling cause issues with existing code?

Reply:
Oh, I see the old accessor methods spelled it this way!

Reply:
Yep! I renamed the variable so the getter/setter methods would be the same as before.

```java
  public final JavaScript getJavaScript() {
    return javascript;
  }

  public final void setJavaScript(JavaScript newJavaScript) {
    this.javascript = newJavaScript;
  }
```